### PR TITLE
Test Updates

### PR DESCRIPTION
**Pull request by pranav7**

### CODE REVIEW

File: spec/lib/lightning/api_spec.rb [hunk 1]
```diff
@@ -1,186 +1,184 @@
...
+      retrieved_feature = described_class.get(feature_key)
       expect(retrieved_feature).to eq(created_feature)
     end
   end
 
-  context '.list' do
-    it 'list all features with no results' do
-      expect(Lightning::Api.list).to be_empty
+  describe '.list' do
+    it 'retursn empty list when no features exists' do
```
Comment (ruthwickp) [inline spec/lib/lightning/api_spec.rb]: typo in ```returns```

File: spec/lib/lightning/api_spec.rb [hunk 2]
```diff
@@ -1,186 +1,184 @@
...
-  dummy_user = User.create(name: 'Dummy User')
+RSpec.describe Lightning::Api do
+  let(:user) { User.create(name: 'Dummy User') }
 
-  context '.create' do
+  describe '.create' do
     it 'creates the feature' do
-      Lightning::Api.create!('create_feature', 'New feature for testing')
-      expect(Lightning::Api.get('create_feature')).to be_an_instance_of Lightning::Feature
+      described_class.create!('create_feature', 'New feature for testing')
```
Comment (ruthwickp) [inline spec/lib/lightning/api_spec.rb:10]: I'm curious where this `described_class` is defined

Comment (pranav7) [reply spec/lib/lightning/api_spec.rb:10]: It is an rspec construct: https://relishapp.com/rspec/rspec-core/docs/metadata/described-class
Basically, it refers to the class described in the first line, in the future if we decide to change the definition of the class, we won't have to replace every instance.

File: spec/lib/lightning/api_spec.rb (hunk 1 above)
Comment (pranav7) [inline spec/lib/lightning/api_spec.rb]:
```suggestion
    it 'returns an empty list when no features exist' do
```

**Review by ruthwickp — APPROVED** [review]

**Pull request merged**